spec/functional/resource/group_spec.rb

```diff
@@ -292,14 +292,16 @@ def create_group
     end
   end
 
-  let(:group_name) { "group#{SecureRandom.random_number(9999)}" }
+  # Groups below 1000 are reserved on a number of OSes, pick higher numbers
+  let(:number) { SecureRandom.random_number(9899) + 100 }
+  let(:group_name) { "grp#{number}" } # groupname 8 characters or less
```

Comment:
Can you mention we're doing <= 8 for Solaris

Reply:
done, and a reference added that it's not just solaris with the <= 8 limit